From 7bee7691eec41aa0d3697466250be7680a5e963d Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Thu, 14 Jun 2018 16:43:28 +0500
Subject: [PATCH] kvm.h: workaround kernel version issues

The kernel version for this change is different than
what we currently have for the AMD BSPs as there are
various patches backported for the kernel that make
this change otherwise irrelevant. Fix this by forcing
a known version that works.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 instrumentation/events/lttng-module/kvm.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/instrumentation/events/lttng-module/kvm.h b/instrumentation/events/lttng-module/kvm.h
index 698cc71..840b7de 100644
--- a/instrumentation/events/lttng-module/kvm.h
+++ b/instrumentation/events/lttng-module/kvm.h
@@ -86,7 +86,7 @@ LTTNG_TRACEPOINT_EVENT(kvm_ack_irq,
 
 #if (LINUX_VERSION_CODE >= KERNEL_VERSION(4,15,0) \
 	|| LTTNG_KERNEL_RANGE(4,14,14, 4,15,0) \
-	|| LTTNG_KERNEL_RANGE(4,9,77, 4,10,0) \
+	|| LTTNG_KERNEL_RANGE(4,7,0, 4,10,0) \
 	|| LTTNG_KERNEL_RANGE(4,4,112, 4,5,0) \
 	|| LTTNG_KERNEL_RANGE(3,16,52, 3,17,0) \
 	|| LTTNG_KERNEL_RANGE(3,2,97, 3,3,0))
-- 
2.11.1

